### PR TITLE
(PC-9223) Configure id check app according to settings

### PR DESCRIPTION
**Pull request by BenStirrup**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-9223

Also include:

- https://passculture.atlassian.net/browse/PC-9214

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Written **documentation on Notion** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] |
| -------------------- | :----------------------: |
|                      |                          |

### CODE REVIEW

**Review by kopax-polyconseil — APPROVED** [review]

File: src/ConfiguredIdCheckProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import { useAppSettings } from 'features/auth/settings'
+import { env } from 'libs/environment'
+import { errorMonitoring } from 'libs/errorMonitoring'
+import { idCheckAnalytics } from 'libs/idCheckAnalytics'
+
+export const ConfiguredIdCheckProvider: FunctionComponent = function (props) {
+  const { data: settings } = useAppSettings()
+
+  const stepsConfigs = useMemo(() => {
+    return getStepsConfigs({
```
Comment (kopax-polyconseil) [inline src/ConfiguredIdCheckProvider.tsx:21]: usStepConfig direct, et tu met appSettings dedans, comme ,ca moins de logique dans ce composant et un truc qui fait une chose bien

File: src/ConfiguredIdCheckProvider.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import { t } from '@lingui/macro'
```
Comment (kopax-polyconseil) [inline src/ConfiguredIdCheckProvider.tsx:1]: src trop global , on a pas mieux comme dossier ?

**Review by kopax-polyconseil — APPROVED** [review]

File: src/ConfiguredIdCheckProvider.tsx (hunk 2 above)
Comment (BenStirrup) [reply src/ConfiguredIdCheckProvider.tsx]: yes, j'ai mis tout ce qui concerne idcheck dans un dossier dédié à idcheck

File: src/ConfiguredIdCheckProvider.tsx (hunk 1 above)
Comment (BenStirrup) [reply src/ConfiguredIdCheckProvider.tsx]: Ce n'est plus d'actualité, j'ai mis la logique dans IdCheckV2

**Pull request closed without merging**